snippets/csharp/pipelines/Pipe.cs

```diff
@@ -53,7 +53,7 @@ async Task FillPipeAsync(Socket socket, PipeWriter writer)
             }
 
             // Tell the PipeReader that there's no more data coming.
```

Comment:
> Sounds good to me. You should open a PR.

@halter73 would the suggestion within this PR work?

```suggestion
             // By completing PipeWriter, we tell the PipeReader that there's no more data coming.
```

Reply:
Thanks!